src/coreComponents/physicsSolvers/surfaceGeneration/EmbeddedSurfacesParallelSynchronization.cpp

```diff
@@ -104,7 +103,7 @@ void unpackNewNodes( NeighborCommunicator * const neighbor,
 
   parallelDeviceEvents events;
 
-  unpackedSize += nodeManager.unpackNewNodesGlobalMaps( receiveBufferPtr, newGhostNodes );
+  nodeManager.unpackNewNodesGlobalMaps( receiveBufferPtr, newGhostNodes );
```

Comment:
@wrtobin @CusiniM 
What happened to our checking of unpackSize? Shouldn't we be error checking all of this to make sure we have the right number of bytes unpacked?